pkg/certmanager/cert_manager.go

```diff
@@ -4,12 +4,20 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/asaskevich/govalidator"
 	"github.com/go-logr/logr"
-	"github.com/manifestival/manifestival"
+	mf "github.com/manifestival/manifestival"
+	"github.com/shipwright-io/operator/pkg/common"
 	crdclientv1 "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset/typed/apiextensions/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+)
 
-	"github.com/shipwright-io/operator/pkg/common"
+var (
+	certDomainsTemplate = []string{
+		"shipwright-build-webhook.%s.svc",
+		"shp-build-webhook.%s.svc",
```

Comment:
Same here, I assume we only need shp-build-webhook.

Reply:
same here, I will keep just shp-build-webhook

Reply:
fixed